modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddListTypeEntry.tsx

```diff
@@ -39,16 +39,25 @@ const ModalAddListTypeEntry: React.FC<IProps> = ({
 }) => {
 	const [error, setError] = useState<string>('');
 	const [selectedLocale, setSelectedLocale] = useState<TLocale>(
-		defaultLocale as {
-			label: string;
-			symbol: string;
-		}
+		defaultLocale as TLocale
 	);
 	const initialValues: TInitialValues = {
 		key: undefined,
 		name_i18n: {[defaultLanguageId]: ''},
 	};
 
+	const setLocaleAndClose = (value: TLocale) => {
```

Comment:
Instead of calling it `setLocaleAndClose` wouldn't be better to be `handleLocaleChange`?

Reply:
Thanks, @rebsilva! This is much better because `setLocaleAndClose` could be confusing and looks like a setState.

Reply:
Exactly! Thanks Aline! :heart: